<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSpendingsDialog</class>
 <widget class="QDialog" name="AddSpendingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddSpendingsDialog</string>
  </property>
  <widget class="QLineEdit" name="categoryNameLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Category Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryTranzactionValueLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Tranzaction Value</string>
   </property>
  </widget>
  <widget class="QComboBox" name="categoryFinancePicker">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="financeISOLineEdit">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>150</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>ISO</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryTranzactionDestinationLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>291</width>
     <height>30</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Transaction destination</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="categoryTranzactionDetailsLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="placeholderText">
    <string>Tranzaction Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addSpendingsPushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelAddSpendingsPushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
